contracts/common/bases/StrategyRoleableBaseUpgradeable.sol

```diff
@@ -9,50 +9,66 @@ abstract contract StrategyRoleableBaseUpgradeable is
     AccessControlEnumerableUpgradeable,
     StrategyBaseUpgradeable
 {
-    error MissingDefaultAdminRole();
+    error MissingAdminRole();
     uint256[8] private __gap;
 
     // Manages sensitive parameters of the strategy for example fee swap service.
     // Any parameter change that can result in stolen or lost funds should be managed by the governor role.
-    bytes32 public constant GOVERNOR_ROLE = keccak256("GOVERNOR_ROLE");
+    // keccak256("GOVERNOR_ROLE")
+    bytes32 public constant GOVERNOR_ROLE =
+        0x7935bd0ae54bc31f548c14dba4d37c5c64b3f8ca900cb468fb8abd54d5894f55;
+
     // Manages less sensitive parameters of the strategy for example setting investment limits.
-    bytes32 public constant STRATEGIST_ROLE = keccak256("STRATEGIST_ROLE");
+    // keccak256("STRATEGIST_ROLE")
+    bytes32 public constant STRATEGIST_ROLE =
+        0x17a8e30262c1f919c33056d877a3c22b95c2f5e4dac44683c1c2323cd79fbdb0;
+
     // Manages routine, periodic tasks like delta neutral rebalancing and liquidation protection.
-    bytes32 public constant MAINTAINER_ROLE = keccak256("KEEPER_ROLE");
+    // keccak256("MAINTAINER_ROLE")
+    bytes32 public constant MAINTAINER_ROLE =
+        0x339759585899103d2ace64958e37e18ccb0504652c81d4a1b8aa80fe2126ab95;
+
     // Manages smart contract upgrades.
-    bytes32 public constant UPGRADE_ROLE = keccak256("UPGRADE_ROLE");
+    // keccak256("UPGRADE_ROLE")
+    bytes32 public constant UPGRADE_ROLE =
```

Comment:
Do I understand it correctly, that we could have a different wallet for each of these roles?
Something that isn't present in the current CALM portfolio afaik.

Reply:
yes, and we do need different accounts, as our off-chain component is permissioned, it cannot easily be a multisig